### PR TITLE
Improve Spanish localization

### PR DESCRIPTION
**Pull request by plotters**

### CODE REVIEW

**Review by x64x2 — APPROVED** [review]

File: resources/lang/es/navmenu.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?php
+
+return [
+	'search'			=> 'Buscar',
+	'home'				=> 'Inicio',
+	'local'				=> 'Local',
+	'network'			=> 'Red',
+	'discover'			=> 'Discubrir',
```
Comment (pitbuster) [inline resources/lang/es/navmenu.php]:
```suggestion
	'discover'			=> 'Descubrir',
```

File: resources/lang/es/navmenu.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	'viewMyProfile'		        => 'Ver mi perfil',
+	'myProfile'			=> 'Mi Perfil',
+	'myTimeline'	                => 'Mi cronología',
+	'publicTimeline'	        => 'Cronología pública',
+	'remoteFollow'		        => 'Seguimiento remoto',
+	'settings'			=> 'Configuración',
+	'admin'				=> 'Administración',
+	'logout'			=> 'Cerrar sesión',
+	'directMessages'	        => 'Mensajes directos',
+	'composePost'		        => 'Redactar mensaje',
```
Comment (pitbuster) [inline resources/lang/es/navmenu.php]: Maybe `Crear publicación`, although I am not sure were this translation does in the UI so I don't know if it refers to a media post or not.

**Review by pitbuster — COMMENTED** [review]
Looks good, although I put a comment for a typo and a suggestion for a translation.

**Review by pitbuster — APPROVED** [review]

**Review by x64x2 — APPROVED** [review]

**Comment by dansup** [comment]
Hi, please submit translations to https://translate.pixelfed.org/

**Comment by pitbuster** [comment]
@dansup Spanish is actually 100% translated on crowdin, but only ~30% of translations are approved, any way to contribute there?

**Comment by x64x2** [comment]
> @dansup Spanish is actually 100% translated on crowdin, but only ~30% of translations are approved, any way to contribute there?

Crowdin??

**Comment by pitbuster** [comment]
>Crowdin??

But how? The strings are already there, only lacking approval. It is not clear if there is a way to vote/confirm/move the approval forward in the platform :(

**Pull request closed without merging**